target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -40,8 +40,11 @@ alfa-network,n5q)
 	ucidef_set_led_rssi "signal3" "SIGNAL3" "green:signal3" "wlan0" "50" "100"
 	ucidef_set_led_rssi "signal4" "SIGNAL4" "green:signal4" "wlan0" "75" "100"
 	;;
-alfa-network,pi-wifi4)
-	ucidef_set_led_netdev "lan_data" "LAN_DATA" "orange:lan_data" "eth0" "tx rx"
+alfa-network,pi-wifi4|\
+watchguard,ap100|\
+watchguard,ap200|\
+watchguard,ap300)
+	ucidef_set_led_netdev "lan_data" "LAN_DATA" "orange:lan_data" "eth0" "rx"
```

Comment:
Why do you change this from `"tx rx"` to `"rx"` ?

Reply:
It's not very important, but...

if the other host is dead (powered link only), or if the ethernet is not configured properly (frames failing checksum) then "tx" blinking still occurs, but is not an indicator of real traffic making it from one side to another. it is only "rx" that tells you that real data is making it from the physical layer to the mac layer.

the amount of blinking is about the same in normal use with common traffic (but I suppose UDP-only traffic would not blink). it's useful in testing ethernet switch configuration.

if you think something like that should be changed in a separate commit, ill remove it here